ckanext/aircan_connector/action.py

```diff
@@ -66,6 +69,8 @@ def datapusher_submit(context, data_dict):
         '''
 
         table_schema = ckan_resource.get('schema')
+        if not table_schema:
+            raise ValueError(NO_SCHEMA_ERROR_MESSAGE)
```

Comment:
@mbeilin Here, can't you return 
`return {"success": False, "errors": [NO_SCHEMA_ERROR_MESSAGE]}` without throwing the exception?

Reply:
I tried this already, I need to prevent `resource_create` to complete successfully in case the schema is not in request and just 
return {"success": False, "errors": [NO_SCHEMA_ERROR_MESSAGE]} without throwing the exception doesn't help here...